integrations/timescaledb/timescaledb-metrics/CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog][changelog] and this project adheres
+to [Semantic Versioning][semver].
+
+## Unreleased
+
+- *n/a*
+
+## [0.1.0]
+
+- Initial add of `timescaledb-metrics` integration.
```

Comment:
```suggestion
## Unreleased

- Initial add of `timescaledb-metrics` integration.
```

Reply:
We still haven't fully dialed in the release process yet, but we do know we're going to version these with a date-style numbers, e.g. `20220304.0.0`. For now let's init changelogs with "unreleased" comments.